docs/README.md

```diff
@@ -24,6 +24,22 @@
 
 ## Overview
 
+### Quickstart
+
+1. First off, let's kick the tyres and look at the sample _Repospective_ app that
+  comes bundled with this project to demo some of its best features:
+
+    ```Shell
+    npm install && npm start
+    ```
+
+  And go to [localhost:3000](http://localhost:3000) to see it in action.
+
+  - Add a Github username to see Redux and Redux Sagas in action
+  - Click the Features button to see React Router in action
+  - Edit the file at `/app`
```

Comment:
This line is confusing, it seems as if one is prompted to go to `localhost:3000/app`, which isn't actually anything